app/assets/javascripts/misc/i18n-strings.js.erb

```diff
@@ -35,7 +35,10 @@ window.LoginGov = window.LoginGov || {};
   'zxcvbn.feedback.Capitalization doesn\'t help very much',
   'zxcvbn.feedback.All-uppercase is almost as easy to guess as all-lowercase',
   'zxcvbn.feedback.Reversed words aren\'t much harder to guess',
-  'zxcvbn.feedback.Predictable substitutions like \'@\' instead of \'a\' don\'t help very much'
+  'zxcvbn.feedback.Predictable substitutions like \'@\' instead of \'a\' don\'t help very much',
+  'shared.time.minute',
+  'shared.time.second',
+  'shared.grammar.and',
```

Comment:
I'm not seeing where `and` is used? And similarly to plurals, not all languages handle things the same way, so I'd recommend removing it if possible?